Directory.Build.targets

```diff
@@ -0,0 +1,3 @@
+<Project>
+  <Sdk Name="Microsoft.Build.CentralPackageVersions" Version="2.0.1" />
```

Comment:
The version of SDK should be provided only in `global.json` file. I think we can remove it from here.